mugur.el

```diff
@@ -1150,7 +1158,9 @@ that mugur-key."
   ((DF base)       "DF(BASE)")
   ((DF "base")     "DF(BASE)")
   ((LM "base" C)   "LM(BASE, MOD_LCTL)")
-  ((LT "base" x)   "LT(BASE, KC_X)")))
+  ((LT "base" x)   "LT(BASE, KC_X)")
+  ((RAW "custom_td_fn" "custom_td_fn"))
+  ((RAW "KC_ASUP")   "KC_ASUP")))
```

Comment:
We can, of course, add KC_ASUP to `mugur--symbol`, right?

Reply:
We can add all the keycodes that are missing, yes. There's another PR that I'll eventually add with a few missing ones.